pywinauto/base_application.py

```diff
@@ -763,9 +763,9 @@ def window(self, **kwargs):
         kwargs['backend'] = self.backend.name
         if kwargs.get('top_level_only') is None:
             kwargs['top_level_only'] = True
-            # XXX: vryabov
-            #if self.backend.name == 'win32':
-            #    kwargs['visible'] = True
+            # TODO: figure out how to eliminate this workaround
```

Comment:
How about overloading the window method in windows.application.Application class? Something like:
```python
def window(self, **kwards):
     kwargs['visible'] = True
     super(Application.self).window(self, **kwargs)
```

Reply:
It will be included into `backend="uia"` implementation. Now it's for `backend="win32"` only. I'd prefer to get red the most of these workarounds. Maybe we can filter out some obviously wrong handles in `children` implementation of `backend="win32"`.